lib/engine/game/g_18_zoo.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+require_relative '../config/game/g_18_zoo'
+require_relative '../g_18_zoo/corporation'
+require_relative '../g_18_zoo/company'
+require_relative 'base'
+
+module Engine
+  module Game
+    module Internal
+      class G18ZOO < Base
+        DEV_STAGE = :alpha
```

Comment:
Please don't set a game to alpha

Reply:
ok, so what stage should I put?

Reply:
when it is playable and you want peolpe to test it

Reply:
If you remove line 12 the game will use the DEV_STAGE from the Base (base.rb) class, which has the value :pre-alpha.
To create an 18Zoo game in your fork, just add ?all to the new game link in your browser, and all pre-alpha games will appear as choices as well.